# Message to control multiple channels of the controller

RoboteqMotorControl[] motors